### DISCUSSION TITLE
Multiple proteases

### DISCUSSION THREAD

**Discussion by ABRai11** [vdemichev/DiaNN#1309] [Q&A]
Hi Vadim,

I have used trypsin and chymotrypsin enzymes for digestion. How do we search in DIA-NN using both enzymes? Is it possible to give a single search?

Thanks,
Akhila

**Comment by vdemichev** [comment]
Hi Akhila,

You can specify cut positions with --cut. Probably makes sense to enable 1 or 2 missed cleavages.

Best,
Vadim

**Comment by ABRai11** [comment]
Hi Vadim,

Thank you for the reply. Is this command correct for trypsin and chymotrypsin "--cut K*,R*,!*P, F*, W*, Y*"

**Comment by Althalis** [comment]
Hi Vadim and ABRai11, 
trying to guess the command for trypsin and Glu-C co-digestion: "--cut K*,R*,!P, D*, E*" ?
I'm not sure where to find details about the meaning of * and !, am I correct guessing * means 'always after' and ! 'never before'?
Any other useful command details to read somewhere? :)
Thank you for your help, 
Best, 
Julie

**Reply by vdemichev** [reply]
Hi Julie,

Please see the command line reference https://github.com/vdemichev/DiaNN?tab=readme-ov-file#command-line-reference for --cut.
So just --cut K*,R*,D*,E*.

Best,
Vadim
